app/templates/events/trackVolunteers.html

```diff
@@ -234,57 +234,55 @@ <h4>RSVP Waitlist</h4>
 <div class="modal fade float-start" id="addVolunteerModal" tabindex="-1" aria-labelledby="addVolunteerModalLabel" aria-hidden="true">
   <div class="modal-dialog">
     <div class="modal-content">
+      <form action="/addVolunteersToEvent/{{event.id}}" method="POST">
       <div class="modal-header">
-        {% set modalTitle = "Add Volunteer to Waitlist" if rsvpFull else("Add Volunteer RSVP" if rsvpNeeded else "Add Volunteer") %}
-        <h5 class="modal-title" id="exampleModalLabel">{{modalTitle}}</h5>
-        <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+        {% if event.isPast %}
+          <h5 class="modal-title">Add Past Volunteers</h5>
+          <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+        {% else %}
+          {% set modalTitle = "Add Volunteer to Waitlist" if rsvpFull else("Add Volunteer RSVP" if rsvpNeeded else "Add Volunteer") %}
+          <h5 class="modal-title" id="exampleModalLabel">{{modalTitle}}</h5>
+          <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+        {% endif %}
       </div>
-      <div class="modal-body">
+      <div class="modal-body"> 
         <div class="input-group">
           <div class="form-group input-group-x form-outline ui-widget" style="width:80%;">
             <input type="input" id="addVolunteerInput" class="form-control" placeholder="Search" autocomplete="off"/>
           </div>
           <button type="button" class="btn btn-primary btn-sm">
             <i class="bi bi-search ms-0"></i>
           </button>
+          <div class="modal-body pl-5">
+            {%for participant in recurringVolunteers%}
```

Comment:
The way this is right now makes all previous volunteers show up in past events no matter what. What we want to have happen is for the previous volunteers to show up when the event is a recurring event and it should only be the volunteers from the previous events in the series.